tests/driver_mhz19/README.md

```diff
@@ -0,0 +1,12 @@
+# MH-Z19 Driver Test
+
+## Introduction
+This test will test if the MH-Z19 C02 gas sensor is working.
```

Comment:
Mention MH-Z19B as well. It works as well

Reply:
Added a small explanation about the MH-Z19B also working